mne/viz/topomap.py

```diff
@@ -1971,60 +1973,59 @@ def plot_epochs_psd_topomap(epochs, bands=None,
         Only use tapers with more than 90%% spectral concentration within
         bandwidth.
     %(normalization)s
-    ch_type : 'mag' | 'grad' | 'planar1' | 'planar2' | 'eeg' | None
-        The channel type to plot. For 'grad', the gradiometers are collected in
-        pairs and the mean for each pair is plotted. If None, then first
-        available channel type from order given above is used. Defaults to
-        None.
-    %(cmap_psd_topo)s
+    %(ch_type_psd_topomap)s
+    %(normalize_psd_topo)s
     %(agg_fun_psd_topo)s
     %(dB_plot_topomap)s
-    %(n_jobs)s
-    %(normalize_psd_topo)s
-    %(cbar_fmt_psd_topo)s
     %(outlines_topomap)s
-    %(axes_plot_topomap)s
-    %(show)s
     %(sphere_topomap_auto)s
+    %(cmap_psd_topo)s
     %(vlim_psd_topo_joint)s
+    %(cbar_fmt_psd_topo)s
+    %(axes_plot_topomap)s
+    %(show)s
+    %(n_jobs)s
     %(verbose)s
 
     Returns
     -------
     fig : instance of Figure
-        Figure distributing one image per channel across sensor topography.
+        Figure showing one scalp topography per frequency band.
     """
-    ch_type = _get_ch_type(epochs, ch_type)
-    units = _handle_default('units', None)
-    scalings = _handle_default('scalings', None)
-    unit = units[ch_type]
-    scaling = scalings[ch_type]
-
-    picks, pos, merge_channels, names, ch_type, sphere, clip_origin = \
-        _prepare_topomap_plot(epochs, ch_type, sphere=sphere)
-    outlines = _make_head_outlines(sphere, pos, outlines, clip_origin)
-
-    psds, freqs = psd_multitaper(epochs, tmin=tmin, tmax=tmax,
-                                 bandwidth=bandwidth, adaptive=adaptive,
-                                 low_bias=low_bias,
-                                 normalization=normalization, picks=picks,
-                                 proj=proj, n_jobs=n_jobs)
-    psds = np.mean(psds, axis=0)
-    psds *= scaling**2
-
-    if merge_channels:
-        psds, names = _merge_ch_data(psds, ch_type, names, method='mean')
-
-    return plot_psds_topomap(
-        psds=psds, freqs=freqs, pos=pos, agg_fun=agg_fun,
-        bands=bands, cmap=cmap, dB=dB, normalize=normalize,
-        cbar_fmt=cbar_fmt, outlines=outlines, axes=axes, show=show,
-        sphere=sphere, vlim=vlim, unit=unit, ch_type=ch_type)
+    # add after dB
+    # %(sensors_topomap)s
+    # %(show_names_topomap)s
+    # %(mask_evoked_topomap)s
+    # %(mask_params_topomap)s
+    # %(contours_topomap)s
+    # add after sphere
+    # %(image_interp_topomap)s
+    # %(extrapolate_topomap)s
+    # %(border_topomap)s
+    # %(res_topomap)s
+    # %(size_topomap)s
+    # add after vlim
+    # %(colorbar_topomap)s
+    # add after cbar_fmt
+    # %(units_topomap)s
```

Comment:
This looks like some TODO but you want to hold off on it until topomap plotting has improved?